libraries/default.rb

```diff
@@ -475,6 +475,10 @@ def configure_cloudwatch_log(log_name, log_file, datetime_format)
       retention_days = node.fetch(:cwlogs_retention_days, '30')
       region = node.fetch(:region, 'us-east-1')
 
+      service 'awslogs' do
```

Comment:
without this the `template` block further down doesn't know how to notify the awslogs service to restart. This is OK when called in a  chef run that includes the `install-cwlogs` recipe, but fails in other contexts, e.g., a one-off run of a recipe that tries to use `configure_cloudwatch_log`.